### PR TITLE
Add missing metrics to RefMan

### PR DESCRIPTION
**Pull request by jbartok**
Add description here

Checklist
- [x] Tags Set
- [x] Milestone Set

### CODE REVIEW

File: hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc [hunk 1]
```diff
@@ -178,6 +178,13 @@ Additionally, if the vertex sourcing them is a data source or data sink, then th
 |===
 |Names|Extra tags
 
+|**executionStartTime**: Start time of the current execution of a job. +
+**executionCompletionTime**: Start time of the current execution of a job. +
+
+Both contain epoch time in milliseconds.
+|** * only ones without `vertex` tag** +
```
Comment (cangencer) [inline hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc]: what does this mean?

Comment (jbartok) [reply hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc]: See the first paragraph of the "Jet Job metrics" section, stating that almost all job metrics have the 'vertex' tag set (ie. are per vertex). So I wanted to point out that these two metrics are the exceptions.

Comment (jbartok) [reply hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc]: Pushed a more straightforward version.

File: hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc [hunk 2]
```diff
@@ -178,9 +178,16 @@ Additionally, if the vertex sourcing them is a data source or data sink, then th
 |===
 |Names|Extra tags
 
+|**executionStartTime**: Start time of the current execution of the job. +
+**executionCompletionTime**: Start time of the current execution of the job. +
```
Comment (viliam-durina) [inline hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc]: copy-paste error

File: hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc [hunk 3]
```diff
@@ -178,9 +178,16 @@ Additionally, if the vertex sourcing them is a data source or data sink, then th
...
+
+Both contain epoch time in milliseconds.
+|** no extra tags, they are even missing `vertex`** +
```
Comment (viliam-durina) [inline hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc]: `they are even missing 'vertex'` - no surprise here. I personally would just skip this part, the next sentence is enough,

**Comment by jbartok** [comment]
> I wonder what the value is when the job is running.

There is no completion time metric value when the job is running (see `ExecutionContext.provideDynamicMetrics`, the metric gets set by dynamic collection only if there is a non-zero value).

Actually the only way to get/see a value for this execution completion metric is to set `JobConfig.storeMetricsAfterJobCompletion` and then use the Job Metrics API (after completion).

**Pull request merged**